addons/mail/static/src/models/composer/composer.js

```diff
@@ -78,6 +78,19 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * @private
+         * @returns {mail.partner[]}
+         */
+        _computeRecipients() {
+            const mentionned = this.mentionedPartners.map(partner => partner.id);
```

Comment:
Should be a list of records instead of ids?

Strange that runbot is green for mentions.